frontend_tests/node_tests/narrow_state.js

```diff
@@ -23,6 +24,15 @@ function set_filter(operators) {
 
     assert(!narrow_state.is_for_stream_id(test_stream.stream_id));
 
+    var bad_stream_id = 1000000;
+    var called = false;
+    global.blueslip.error = function (msg) {
+        assert.equal(msg, 'Bad stream id ' + bad_stream_id);
+        called = true;
+    };
+    assert(called);
```

Comment:
This should be after the next line, because right now it's just checking `assert(false)`, so the test will fail.

Reply:
Thanks @sampritipanda for catching this!